Tasks/AzureAppServiceSettingsV1/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,2248 @@
+
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+Azure App Service Deploy incorporates third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
+
+1.	abbrev (https://github.com/isaacs/abbrev-js)
```

Comment:
Please verify is this is correct or has some packages listed that you dont use in task.